packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -286,6 +324,9 @@ export default {
                 this.dropbox.draggingOver = false
             }
         }
+    },
+    mounted() {
+        window.setInterval(this.updateVerificationItems, 10000)
```

Comment:
I would only check the items which are not in the desired state. I guess we only refresh for items which are in some kind of processing state and not to check the switch from registered to revoked.

Reply:
I just assumed it's a general auto refresh but maybe you are right. @emeka What do you think?

Reply:
I would do a generic auto refresh as it could be used to know the state change of a file.  For example I could keep the browser window open and see that some one else signed the document (multi-sign).  In addition, it avoids having to know which states must be refreshed and which not.

Reply:
Changed as discussed